Configuration/StandardSequences/python/RawToDigi_cff.py

```diff
@@ -102,14 +102,17 @@
 ctpps_2016.toReplaceWith(RawToDigi_noTk, _ctpps_2016_RawToDigi_noTk)
 
 # GEM settings
-_run2_GEM_2017_RawToDigi = RawToDigi.copy()
-_run2_GEM_2017_RawToDigi.insert(-1,muonGEMDigis)
+_gem_RawToDigi = RawToDigi.copy()
+_gem_RawToDigi.insert(-1,muonGEMDigis)
 
-_run3_RawToDigi = _run2_GEM_2017_RawToDigi.copy()
-_run3_RawToDigi.insert(-1,muonME0Digis)
+_me0_RawToDigi = _gem_RawToDigi.copy()
```

Comment:
(As above) it seems safer to me not to make the different settings depend on each other.
Therefore i would copy() here RawToDigi once again, instead of an already modified version of it, and insert() the muonGEMDigis and muonME0Digis on it